### DISCUSSION TITLE
Welcome to CatalogBuilder Discussions!

### DISCUSSION THREAD

**Discussion by aradhakrishnanGFDL** [NOAA-GFDL/CatalogBuilder#21] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by apreucil** [comment]
My name is Anthony Preucil and I am a physical scientist at GFDL. I am looking forward to improving catalogs so that the GFDL user community can more easily perform analysis and create workflows that are fast, easy and reproducible.

**Reply by ceblanton** [reply]
Welcome! fast, easy, reproducible, indeed. And portable, interoperable, maintainable, and also hopefully also FAIR (https://www.go-fair.org/fair-principles/).
